internal/api/auth/basic_auth.go

```diff
@@ -45,6 +46,19 @@ func NewBasicAuth(cfg config.AuthConfig) (BasicAuth, error) {
 	}
 	b.mode = ldapMode
 
+	// trying a connection to LDAP to check the configuration
+	conn, err := ldap.connect()
+	if err != nil {
+		return b, fmt.Errorf("test LDAP connection: %w", err)
```

Comment:
I am not sure if we want to exit here in case of any error. I am not even sure that we want to have a sanity check here, as the logic afterwards allows to have LDAP connection problems

Reply:
yep you are right, done